src/util.c

```diff
@@ -212,10 +212,12 @@ void print_footer(OSQPInfo* info,
   c_print("status:               %s\n", info->status);
 
   if (polishing && (info->status_val == OSQP_SOLVED)) {
-    if (info->status_polish == 1) {
+    if (info->status_polish == OSQP_POLISH_SUCCESS) {
       c_print("solution polishing:   successful\n");
     } else if (info->status_polish < 0) {
       c_print("solution polishing:   unsuccessful\n");
+    } else if (info->status_polish == OSQP_POLISH_NO_ACTIVE_SET) {
+      c_print("solution polishing:   skipped\n");
```

Comment:
I think calling this "skipped" could cause confusion.   If polishing is disabled as an option then it is also "skipped", and user might reasonably ask why the polishing step was skipped when they had it enabled.

Maybe "not needed" would be clearer here?

Reply:
Should we then change the reported status to be `OSQP_POLISH_NOT_NEEDED` instead of saying `OSQP_POLISH_NO_ACTIVE_SET` to make it clear in the error code as well? (the current one is advantageous because it says why we didn't polish, but I guess on reading it it isn't actually clear we didn't polish).

Reply:
Cleaned up in most recent commit.